client/browser/src/libs/bitbucket/style.scss

```
@@ -19,3 +19,45 @@
         display: block;
     }
 }
+
+.command-palette-popover__bitbucket-server {
```

This is a modifier, not a subelement
```suggestion
.command-palette-popover--bitbucket-server {
```